Workflows/Architecture review.md

```diff
@@ -26,9 +26,30 @@ Please follow this timeline:
 8. Once you've finished the database design, it has to be approved by someone from the management.
 9. Once your architecture has been approved, you can generate a new Rails project. Read more about that on the next page.
 
+Ask the project manager to give you the project specification.
```

Comment:
This seems out of place

Reply:
This should have been handed down way before this 😄 Just ✂️ it